tracker/cli/setup.py

```diff
@@ -0,0 +1,96 @@
+from click import echo, option
+from config import basedir, TRACKER_PASSWORD_LENGTH_MIN, TRACKER_PASSWORD_LENGTH_MAX
+from getpass import getpass
+from re import match, IGNORECASE
+from pathlib import Path
+from os.path import isfile
+from os import rename
+from . import cli
+
+
+@cli.group()
+def setup():
+    """Setup and bootstrap the application."""
+    pass
+
+
+@setup.command()
+@option('--purge', is_flag=True, help='Purge all data and tables.')
+def bootstrap(purge=False):
+    """Bootstrap the environment."""
+
+    from tracker import db
+
+    def mkdir(path):
+        Path(path).mkdir(parents=True, exist_ok=True)
+
+    mkdir('{}/pacman/cache'.format(basedir))
+    mkdir('{}/pacman/log'.format(basedir))
+    mkdir('{}/pacman/arch/x86_64/db'.format(basedir))
+
+    if isfile('{}/app.db'.format(basedir)):
+        rename('{}/app.db'.format(basedir),
+               '{}/tracker.db'.format(basedir))
```

Comment:
Might be worth it to add a comment that this is for backwards compatibility.

Reply:
added comment